sympy/vector/coordsysrect.py

```diff
@@ -98,6 +119,27 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        if transformation is None:
+            transformation = Tuple(rotation_matrix, location)
+
+        if isinstance(transformation, Tuple):
```

Comment:
Why that? 
In this case, we are not able to set `transformation` as equations with variables. Previously, we could do that. For example:
```python
a = CoordSys3D('a', transformation=Tuple(((x*cos(y), x*sin(y), z)), (x, y, z))) 
```
But now you assumed that in following manner we're providing rotation and translation transformations. 
Look at the code in `_connect_to_cartesian` method, lines `363:367`